src/api/utils/initial_data.py

```diff
@@ -3,7 +3,7 @@
 This contains the sample initial data required for the test run of the system.
 """
 from api.models import (ActivityType, Activity, Country, LoggedActivity, Society,
-                        User)
+                        User, Role)
```

Comment:
No Cohort? Definitely missed something. Consider using the initial_data on `ft-log-activities`.